### PR TITLE
ignore logs from Rollbar within RollbarHandler

### PR DESCRIPTION
**Pull request by ezarowny**
Should solve an infinite loop when a connection cannot be established to rollbar.com.

### CODE REVIEW

File: rollbar/logger.py [hunk 1]
```diff
@@ -68,6 +66,9 @@ def setHistoryLevel(self, level):
         logging.Handler.setLevel(self, level)
 
     def emit(self, record):
+        if record.name == 'rollbar':
```
Comment (brianr) [inline rollbar/logger.py]: For good measure, how about if we also hardcode the name of the logger in `rollbar/__init__py` to be the string `"rollbar"`? (Will make it more greppable.)

Comment (ezarowny) [inline rollbar/logger.py]: That should work just fine I'd imagine.

Comment (ezarowny) [inline rollbar/logger.py]: cc @coryvirok

Comment (ezarowny) [inline rollbar/logger.py]: Added a constant for the pyrollbar log name.

**Comment by coryvirok** [comment]
👍

File: rollbar/logger.py [hunk 2]
```diff
@@ -68,6 +66,9 @@ def setHistoryLevel(self, level):
         logging.Handler.setLevel(self, level)
 
     def emit(self, record):
+        if record.name == rollbar.__log_name__:
```
Comment (brianr) [inline rollbar/logger.py]: Can you add a comment explaining why this code is here?

Comment (ezarowny) [inline rollbar/logger.py]: That would be wise :D

**Pull request merged**